app/views/content/teach-disabled-pupils-and-pupils-with-special-educational-needs.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Teach disabled pupils and pupils with special educational needs"
+title: "Teach pupils with special educational needs and disabilities"
```

Comment:
@EllieNodder 

Is it worth putting `(SEND)` at the end of the h1 at least (ie by having a separate `heading`), now that we have changed the wording to the more common phrase?
eg https://www.gov.uk/children-with-special-educational-needs

Looking in GSC, SEND does appear in search terms - not sure what you think about adding it to `title` as well as the h1 - I appreciate this will make the title even longer!

Reply:
good shout - shall we just add it to the h1?